bool initialGuessPose
---
std_msgs/Bool execution
